docs/extension/guides/set-wallet-recovery.md

```diff
@@ -0,0 +1,63 @@
+---
+sidebar_label: '👛 Set Wallet Recovery'
+sidebar_position: 6
+description: 'How to set up wallet recovery on the LUKSO Universal Profile Extension?'
+---
+
+# Set Wallet Recovery
+
+If you want a personal backup solution for your Universal Profile, you can add another wallet as a recovery controller.
+
+## Preparation
+
+To add a new controller account to your Universal Profile, you will need the [Universal Profile Browser Extension](https://chromewebstore.google.com/detail/universal-profiles/abpickdkkbnbcoepogfhkhennhfhehfn) as well as an [Ethereum wallet with custom network support](../../general/wallet-support.md) installed **in the same [supported browser](../introduction.md#which-browsers-does-the-extension-support)**.
+
+:::tip Wallet Support
+
+The [Wallet Recovery Setup](https://my.universalprofile.cloud/3rd-party/add-recovery) supports **MetaMask** as the wallet. If you experience issues with other wallets, please try again using MetaMask.
```

Comment:
We'll also "promote" Ledger and Coinbase wallet. Maybe later we'll add more logos